src/ui/ExportDialog.tsx

```diff
@@ -10,6 +10,11 @@ import Button from './components/Button';
 import PropertyGrid, { createDefaultObject } from './components/PropertyGrid';
 import Setting, {Text, DropDown, ExtraButton, Toggle} from './components/Setting';
 
+export const templateOptions = [
```

Comment:
Hey man, you shouldn't modified this file. when `optionsMeta` definded a dropdown, the PropertyGrid will automatically render dropdown.

Please double check my leatest code.